src/components/home/vacations-card.tsx

```diff
@@ -143,6 +143,7 @@ const VacationsCard = () => {
         setError(`${strings.vacationRequestError.fetchRequestError}, ${error}`);
       }
     }
+    setLoading(false);
```

Comment:
I agree with updating the loading state here but you can now remove the setLoading(false) from within the try block.